### DISCUSSION TITLE
Welcome to quicktime_video_hack Discussions!

### DISCUSSION THREAD

**Discussion by danielpaulus** [danielpaulus/quicktime_video_hack#123] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ashwani360** [comment]
I was trying to run command qvh gstreamer --udid=138933156687d36dc1d420 but my iPhone screen is not getting stream and seeing below error
{"level":"info","msg":"Starting Gstreamer..","time":"2022-09-09T12:26:24+05:30"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x4147df6]

goroutine 1 [running]:
github.com/danielpaulus/quicktime_video_hack/screencapture/gstadapter.setUpVideoPipeline(0xc00009a9b0, 0x10)
	/Users/danielpaulus/privaterepos/quicktime_video_hack/screencapture/gstadapter/gst_pipeline_builder_mac.go:17 +0x96
github.com/danielpaulus/quicktime_video_hack/screencapture/gstadapter.New(0xc0000b0000)
	/Users/danielpaulus/privaterepos/quicktime_video_hack/screencapture/gstadapter/gst_adapter.go:35 +0xa1
main.startGStreamer(0xc0000b6007, 0x28)
	/Users/danielpaulus/privaterepos/quicktime_video_hack/main.go:218 +0x7a
main.main()
	/Users/danielpaulus/privaterepos/quicktime_video_hack/main.go:129 +0x835


Can any one help me with this
